src/__tests__/stateFromElement-test.js

```diff
@@ -9,23 +9,24 @@ import {
 import {convertToRaw} from 'draft-js';
 import {jsdom} from 'jsdom';
 import fs from 'fs';
+import {EOL} from 'os';
 import {join} from 'path';
 
 let document = jsdom('<!doctype html><html><body></body></html>');
 
 // This separates the test cases in `data/test-cases.txt`.
-const SEP = '\n\n#';
+const SEP = `${EOL}${EOL}#`;
```

Comment:
I do not believe this has anything to do with the operating system's EOL characters. It's simply the hard-coded separator in the test file. This should not change no matter which OS you're using.